src/pychart/runner.py

```diff
@@ -1,12 +1,24 @@
-from ..scanner import Scanner
+from src.pyparser import Parser
+from src.scanner import Scanner
 
 
 def run(source: str):
     scanner = Scanner(source)
     tokens = scanner.get_tokens()
+    ast = Parser(tokens).parse()
+
+    if ast is None:
+        return
+
+    value = ast()
+
+    try:
+        value = int(value)
+    except:
+        pass
```

Comment:
Weird fuckery, should be removed once we figure out how floats/ints/numbers work in out language